docs/resources/trigger.md

```diff
@@ -194,7 +194,7 @@ Example:
 ```
 
 `actions` argument shares the available conditions which can be applied. The conditions are:
-  - `action_id` - (Required) The type of action when triggered.
+  - `action_id` - (**required**) the type of action when triggered.
     * `send_notification` - Send notification to existing integration resource.
     * `block_ips` - Block indicated IP addresses.
```

Comment:
mention that it is connected with denylist

Reply:
what are indicated IP addresses?

Reply:
It must be a mistake. A trigger's `block_ips` action means "block the IP addresses from which the requests originated".